### DISCUSSION TITLE
SOC_Source is referencing a different node than Host.name of node receiving traffic

### DISCUSSION THREAD

**Discussion by bicfordio** [Security-Onion-Solutions/securityonion#3451] [Unsupported Versions] [answered]
I have a distributed setup with a manager node that is communicating with a few Heavy nodes hosted across the US. Each Heavy node is hosted upon a dedicated server with plenty of specifications to host the node. My distributed setup is running on version 2.3.30. 

Something I have found extremely odd is that whenever an alert is generated, the SOC_Source being referenced is consistently different from the host that is actually receiving the traffic that the alert is generated upon. I tested to ensure that there is no cross-communication between the nodes by running a tcpdump off the management interface and did not see any cross traffic. The only linkage between each node is the manager node itself.

Has anyone else noticed this? Is this any reason for concern, or should it just be ignored?

**Comment by weslambert** [comment, marked as answer]
`soc_source` references the data source from an Elasticsearch perspective, as in which search node/index is backing the record viewable in SOC.  If you would like to see where the event came from (for example, which sensor, etc), you'll want to look for `observer.name`.
